### PR TITLE
Portfolio: desktop version

### PR DESCRIPTION
**Pull request by JansueT**
### Project requirements

-  I make sure that mine dekstop version looks like the one in the Figma design template 3 that I chose. I include a screenshot of Template 3 below for reference ;
  
Template 3-main

- I use media queries in my CSS file to make my desktop version look like the template I selected (Desktop: 768px or wider).
- In order to lay out the elements on the page I  use Grid and Flexbox.
- I don't implement any functionality that requires JavaScript.
- I replicate the design as much as possible (e.g., font, colors, images, text, margins) using [the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -21,6 +21,21 @@
...
+                </li>
+                <li>
+                    <a class="black" href="#">Contact</a>
+                </li>
+            </ul>
+        
+        
         </nav>
     </header>
     <section class="headline">
```
Comment (Gambit142) [inline index.html:41]: - [ ] For the `headline` section, kindly make your design look identical to that of the Figma template you chose. To make it look identical, kindly work on:

- The background images used
- The `hx` elements with texts `"Hey there. I’m Cansu"` and `"I'm a software developer"`
- The dropdown icon used 💻 

### Your Design

image

### Figma Template

image

File: index.html [hunk 2]
```diff
@@ -313,10 +328,10 @@ <h2>
...
-                <li><button class="btn2">JavaScript</button></li>
-                <li><button class="btn2">Ruby</button></li>
-                <li><button class="btn2">Html</button></li>
-                <li><button class="btn2">CSS</button></li>
+                <li><button type="button" class="btn2">JavaScript</button></li>
+                <li><button type="button" class="btn2">Ruby</button></li>
+                <li><button type="button" class="btn2">Html</button></li>
+                <li><button type="button" class="btn2">CSS</button></li>
             </ul>
         </div>
```
Comment (Gambit142) [inline index.html:336]: - [ ] For the recent `About Myself` section, kindly make this section look identical to that of the Figma template you chose. To highlight the changes to be made in this section, you need to:

- Make the `Languages`, `Framework` and `Skills` subsections align on a row. 
- Use the background images used for this section and align the accordingly
- Alignment and structure of the `p` element with the text `"I can help you build a product ..."`
- The alignment of each card and its badges. 👍 

### Your Design

image

### Figma Design

image


#### Hint: Use the `grid` to style the section and then `flexbox` to style individual cards 👍

File: index.html [hunk 3]
```diff
@@ -358,7 +373,7 @@ <h2>
                 <input type="text" name="name" class="hover" maxlenght="30" placeholder="Full name" required>
                 <input type="email" name="email" class="hover" placeholder="Email address" required>
                 <textarea type="text" name="text" class="hover" maxlenght="500" placeholder="Enter text here" required></textarea>
-                <button class="btngreen">Get in touch</button>
+                <button type="button" class="btngreen">Get in touch</button>
             </form>
         </div>
         <div class="img2"></div>
```
Comment (Gambit142) [inline index.html:379]: - [ ] For the recent `Form` section, kindly make this section look identical to that of the Figma template you chose. To highlight the changes to be made in this section, you need to:

- The `form` and the `p` elements should be on a row. The `form` element should be on the right side while the `p` element with the text `"I'm always interested in hearing ..."` should be on the left side. 💻 
- The background images used should be present and well aligned
- The social media icons are at the center and aligned with the correct margin between each icon 👍 

### Your Design

image

### Figma Design

image

#### P.S: The form input for the full name is two (first and last name) for the desktop version of the design on Figma. However, due to the requirement, you should have one input for the full name, so you do not have to implement it the same way it is implemented on Figma.

image

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @JansueT 👋, 

Good job so far!

## Highlights
- All linters checks passed ✔️ 
- README.md file is professional ✔️ 
- Pull request title and description is descriptive ✔️ 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

#### Please note that when you are implementing the changes I suggested for this milestone, your mobile version should remain intact and identical to the Figma template that you chose 👍 

_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please, remember to tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by JansueT** [comment]
I made all of the changes which requested from the previous code viewer.

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
### Hi @JansueT, 👋 

# STATUS: Changes Required :recycle:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 
-  Has a proper Pull Request. ✔️  
- No Linter errors :heavy_check_mark: 
- Good use of Github flow ✔️ 

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

- [x] **[OPTIONAL but Highly recommended] Great effort working on your portfolio project 👏 👏 👏 , I must commend that you have done amazingly well. But I could realize that while make those effort  working on your `desktop version` , you have introduced a lot of errors in your `mobile version` which has made it distrupted and not well `aligned` with the `mobile screen size` of  `375px`. I would suggest you kindly Fix this as the goal of this `MILESTONE` is to have your `portfolio` project working perfectly on both `mobile screen` as well as `desktop screen`. Thanks and see you.**

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by JansueT** [comment]
Screenshot 2022-12-18 001942

@brainconnect93 Thank You for the code review.

- I did all of requested changes.

File: style.css [hunk 4]
```diff
@@ -377,3 +490,47 @@ hr {
...
+    display: none;
+  }
+
+  .headline {
+    background-image: url(images/header-illsutration-mobile.png);
+  }
+
+  .enginn {
+    flex-direction: column;
+  }
```
Comment (MeqdamAlqudah) [inline style.css:535]: - [Optional] Please make sure that the web page is responsive look how it looks like when  the screen size is changed, I suggest you use more responsive sizing, like VW (try to use one responsive size in all the projects like vw for all the page) which represent the size depending on the screen width   I suggest you take a look at this [website](https://web-development.space/tools/px-to-vw/) compare the size in the design using this [website](https://web-development.space/tools/px-to-vw/)
- 
image

**Review by MeqdamAlqudah — APPROVED** [review]
Hi @hakichris , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by TuyishimireEric** [comment]
# Status: APPROVED 🟢  
### Hi  @hakichris  👋 👋 

#### I'm @TuyishimireEric, Your code reviewer,
Good job so implementing the project requirements! 🥇 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it 🦾  
Congratulations! 🎉 

## Highlights
✔️professional PR👍 
✔️professional README👍 
✔️Codes are clean👍 

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mckent05 — COMMENTED** [review]
Hi @JansueT, 

# STATUS: APPROVED ✔️ ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and well done! 🎉 
approved


## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👏 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**